src/transformers/tokenization_utils_base.py

```diff
@@ -518,7 +519,9 @@ def token_to_chars(self, batch_or_token_index: int, token_index: Optional[int] =
         else:
             batch_index = 0
             token_index = batch_or_token_index
-        return CharSpan(*(self._encodings[batch_index].token_to_chars(token_index)))
+        span_indices = self._encodings[batch_index].token_to_chars(token_index)
+
+        return CharSpan(*span_indices) if span_indices else None
```

Comment:
```suggestion
        return CharSpan(*span_indices) if span_indices is not None else None
```
We are usually explicit in our tests.